CMakeLists.txt

```diff
@@ -172,6 +173,10 @@ elseif (APPLE)
 		set (INSTALL_INCLUDE_DIR include/vapor)
 	endif ()
 	set (CMAKE_INSTALL_RPATH "@executable_path/../lib")
+    # set (INSTALL_NAME_DIR "${CMAKE_INSTALL_PREFIX}/lib")
```

Comment:
Can this commented out code be removed?

Reply:
I was still doing some testing with this code. I ended up uncommenting it and only enabling it for distributable builds.